CMSgov/cms-mdct-qmr
# MDCT-1961: Fetching on Focus Fix

## Description

@bodnarbm found an issue with how `react-query` re-fetches a query on window focus, which was causing a refresh of "stale" data in the QMR measure forms. Now, we're explicitly skipping that re-fetch.

### How to test

./dev local
- Enter any measure
- Start typing data
- Navigate to a different tab

The data should still be there 🎉 

### Changed Dependencies

N/A

## Code author checklist


## Reviewer checklist (two different people)
